tools/ansible/tasks/wsu/powershell/hns.psm1

```diff
@@ -0,0 +1,511 @@
+# Code from https://github.com/microsoft/SDN/
```

Comment:
Please link to the file.

Does this pick just what we need or did you copy the whole file?

Reply:
I copied the whole file, just changed it to only the functions we needed.

Reply:
I am confused. Did you copy the whole file without any changes or did you just copy the function that we need? Either way it needs to be attributed better and we need to check if the license matches up.

Reply:
I copied the whole file originally and added the MIT license, which is what the file was under, to the top of the file.
I'm under the impression that the Apache 2.0 license is compatible with the MIT license, but thats just what google told me, not a lawyer. In the revisions commit I then removed the parts of the file that we did not need.